tests/base/timer_08.cc

```diff
@@ -0,0 +1,76 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2018 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+
+// test TimerOutput with large section names
+
+#include <deal.II/base/timer.h>
+
+#include <algorithm>
+#include <sstream>
+
+#include "../tests.h"
+
+// burn computer time
+double s = 0.;
+void
+burn(unsigned int n)
+{
+  for (unsigned int i = 0; i < n; ++i)
+    {
+      for (unsigned int j = 1; j < 100000; ++j)
+        {
+          s += 1. / j * i;
+        }
+    }
+}
+
+void
+test(TimerOutput::OutputType output_type)
+{
+  std::stringstream ss;
+  {
+    TimerOutput t(ss, TimerOutput::summary, output_type);
+
+    t.enter_subsection("hi");
+    burn(50);
```

Comment:
Do we actually need that?

Reply:
probably not, will remove.

Reply:
I checked, we do need this.

Reply:
I see. We need to make sure that we are not using scientific notation for this to work.
May I suggest to use
```
     std::this_thread::sleep_for(std::chrono::seconds(1));
```
instead?

Reply:
> I see. We need to make sure that we are not using scientific notation for this to work.

correct. The filtering of timing from the table is primitive and does not work with scientific notation.

Will try `sleep_for()`...

Reply:
@masterleinad sleep_for does not work.

Reply:
For me, that works. Still, let's not bother too much. Thanks for trying!